test/test_helper.rb

```diff
@@ -20,25 +20,29 @@
 require 'resque-retry'
 require dir + '/test_jobs'
 
-# make sure we can run redis-server
-if !system('which redis-server')
-  puts '', "** `redis-server` was not found in your PATH"
-  abort ''
-end
+if !ENV['CI'] || ENV['CI'] != 'true'
```

Comment:
Since `ENV['CI']` will return `nil` when it's not set you can just compare the value to `'true'` for this condition.